multisig/src/lib.rs

```diff
@@ -169,16 +175,43 @@ impl MultiSigContract {
             PromiseOrValue::Value(true)
         }
     }
+
+    pub fn get_request(&self, request_id: RequestId) -> MultiSigRequest {
+        self.requests.get(&request_id).expect("No such request")
+    }
+
+    pub fn list_requests(&self) -> Vec<RequestId> {
```

Comment:
maybe `list_request_ids` or `get_all_request_ids` to indicate that it returns IDs